web/static/js/components/stage_help.jsx

```diff
@@ -0,0 +1,62 @@
+import React from "react"
+import ReactDOM from "react-dom"
+
+import { connect } from "react-redux"
+import { bindActionCreators } from "redux"
+import { actions } from "../redux/retro"
+
+import * as AppPropTypes from "../prop_types"
+import STAGES from "../configs/stages"
+
+export const StageHelp = props => {
+  const handleClick = () => {
+    const { retro, actions: { showStageHelp } } = props
+    showStageHelp(retro)
+  }
+
+  const showIcon = () => {
+    const { retro: { stage } } = props
+    const validHelpStages = [
+      STAGES.PRIME_DIRECTIVE,
+      STAGES.IDEA_GENERATION,
+      STAGES.GROUPING,
+      STAGES.VOTING,
+      STAGES.ACTION_ITEMS,
+      STAGES.CLOSED,
+    ]
+    return validHelpStages.includes(stage)
+  }
+
+  if (showIcon()) {
+    return ReactDOM.createPortal(
+      <div className="portal">
+        <i
+          title="Access Stage Information"
+          className="question circle icon"
+          onClick={handleClick}
+          onKeyPress={handleClick}
+        />
+      </div>,
+      document.getElementById("stage-help-icon")
+    )
+  }
+
+  return null
+}
+
+const mapStateToProps = state => {
```

Comment:
I don't think this component will need to be `connect`ed, as it's parent, the `<RemoteRetro />` component, already has the `retro` state and it can pass it down, right?

Reply:
Yes, it looks like the `retro` state is in the parent - good catch.  But I still need to `dispatch` from the component so I'll need to `connect`.

Reply:
You're right, the actions can get passed down.